docs/changelog.md

```diff
@@ -2,6 +2,79 @@
+* [BUGFIX] Fix issues rendering code blocks in API docs ([#6917](https://github.com/great-expectations/great_expectations/pull/6917))
+* [BUGFIX] Fix list_keys method for TupleS3StoreBackend ([#6901](https://github.com/great-expectations/great_expectations/pull/6901)) (thanks @enagovitsyn)
+* [BUGFIX] Fix rendering issue with api docs ([#6924](https://github.com/great-expectations/great_expectations/pull/6924))
+* [BUGFIX] Render bar graph with boolean values ([#6910](https://github.com/great-expectations/great_expectations/pull/6910)) (thanks @tmilitino)
+* [BUGFIX] Capital one contrib/micdavis/import hotfix ([#6922](https://github.com/great-expectations/great_expectations/pull/6922)) (thanks @micdavis)
+* [DOCS] Adding docstring for `Checkpoint.self_check()` ([#6841](https://github.com/great-expectations/great_expectations/pull/6841))
+* [DOCS] `AbstractDataContext.add_store` docstring ([#6851](https://github.com/great-expectations/great_expectations/pull/6851))
+* [DOCS] Doc Strings for ExpectationSuite Display Methods ([#6856](https://github.com/great-expectations/great_expectations/pull/6856))
+* [DOCS] `DataAssistantResult.get_expectation_suite()` docstring ([#6862](https://github.com/great-expectations/great_expectations/pull/6862))
+* [DOCS] Misc docstrings around `DataAssistant` ([#6866](https://github.com/great-expectations/great_expectations/pull/6866))
+* [DOCS] enable running `invoke docstrings` on select modules ([#6868](https://github.com/great-expectations/great_expectations/pull/6868))
+* [DOCS] Adds docstring for expect_column_distinct_values_to_contain_set ([#6855](https://github.com/great-expectations/great_expectations/pull/6855))
+* [DOCS] Documentation Strings for Metric Domain Types and Metric Function Types ([#6872](https://github.com/great-expectations/great_expectations/pull/6872))
+* [DOCS] added docstrings for the public API ([#6884](https://github.com/great-expectations/great_expectations/pull/6884))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_unique` `validate_configuration` ([#6897](https://github.com/great-expectations/great_expectations/pull/6897))
+* [DOCS] Miscellaneous docstrings for `DataContext` and utils ([#6852](https://github.com/great-expectations/great_expectations/pull/6852))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_of_type` `validate_configuration` ([#6896](https://github.com/great-expectations/great_expectations/pull/6896))
+* [DOCS] Add public API docstring for `JsonSchemaProflier.validate` ([#6900](https://github.com/great-expectations/great_expectations/pull/6900))
+* [DOCS] Exclude DataAssistantRunner.run() ([#6919](https://github.com/great-expectations/great_expectations/pull/6919))
+* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](https://github.com/great-expectations/great_expectations/pull/6879)) (thanks @josectobar)
```

Comment:
Looks like Jose's attribution is still here

Reply:
Good catch. Thanks